src/server/game/Battlegrounds/BattlegroundMgr.cpp

```diff
@@ -923,7 +923,8 @@ void BattlegroundMgr::RemoveBattleground(BattlegroundTypeId bgTypeId, uint32 ins
 
 void BattlegroundMgr::InviteGroupToBG(GroupQueueInfo* ginfo, Battleground* bg, TeamId teamId)
 {
-    ASSERT(!ginfo->IsInvitedToBGInstanceGUID);
```

Comment:
Do you understand why an ASSERT was used here?

Reply:
No

Reply:
for zero check?